utils/validation/program.ts

```diff
@@ -0,0 +1,25 @@
+import { Program } from "models/Program";
```

Comment:
@mahadzaryab1 Thoughts on where we should place validation files? Do you think it should be in another directory or does `utils` work?